fivem-discord-scripts/policeemsactivity.md

```diff
@@ -81,6 +93,18 @@ Duration of minutes on tour is now included in the webhook messages! Example sho
 
 ![](https://i.gyazo.com/70c849fce1be1d54c9ccd822744a1ae3.png)
 
+**VERSION CHANGEME**
```

Comment:
Probably version 3.0 since it's changing quite a bit of code. Major changes are usually indicated by a new version number. Minor versions usually something like `2.1`